assets/i18n/en_US.json

```
@@ -327,5 +329,34 @@
         "integrationsContributors": "Integrations contributors",
         "cliContributors": "CLI contributors",
         "managerContributors": "Manager contributors"
+    },
+    "installErrorDialog": {
+        "mount_version_mismatch": "Version mismatch",
+        "mount_no_root": "No root access",
+        "mount_missing_installation": "Installation not found",
+
+        "status_failure_blocked": "Installation blocked",
+        "install_failed_verification_failure": "Verification failed",
+        "status_failure_invalid": "Installation invalid",
+        "install_failed_version_downgrade": "Can't downgrade",
+        "status_failure_conflict": "Installation conflict",
+        "status_failure_storage": "Installation storage issue",
+        "status_failure_incompatible": "Installation incompatible",
+        "status_failure_timeout": "Installation timeout",
+        "status_unknown": "Installation failed",
+
+        "mount_version_mismatch_description": "To install by mounting the patched app, the version of the installed app must match the version of the patched app.\n\nInstall the version of the app you are mounting and try again.",
+        "mount_no_root_description": "To install by mounting the patched app, root access is required.\n\nGrant root access to ReVanced Manager and try again.",
```

```suggestion
        "mount_no_root_description": "To install by mounting the patched app, root access must be granted.\n\nGrant root access to ReVanced Manager and try again.",
```

more consistent with the other strings